### PR TITLE
Added sidebar coding

### PR DESCRIPTION
**Pull request by potnurusrilatha**
please check it

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,9 +3,23 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Zoo</title>
```
Comment (jasminedns) [inline index.html]: I'd name it Homepage

File: styles/index.css [hunk 2]
```diff
@@ -0,0 +1,27 @@
+body {
+    font-family : Zilla slab;
```
Comment (jasminedns) [inline styles/index.css]: I'm not sure the font is being used, you probably need  to implement a @font-face. The rest looks good, goodjob!

**Review by jasminedns — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (potnurusrilatha) [reply index.html]: Corrected reviwed chages

File: styles/index.css (hunk 2 above)
Comment (potnurusrilatha) [reply styles/index.css]: corrected reviwed changes

File: styles/index.css [hunk 3]
```diff
@@ -0,0 +1,29 @@
+body {
```
Comment (jasminedns) [inline styles/index.css:1]: koho font

File: styles/index.css (hunk 2 above)
Comment (potnurusrilatha) [reply styles/index.css]: changed zilla_slab to koho

**Review by jasminedns — APPROVED** [review]
For me is good to go!

**Review by TobiasKjernell — APPROVED** [review]

**Pull request merged**